lib/type.ts

```diff
@@ -45,9 +45,8 @@ class TypeTranspiler extends base.TranspilerBase {
         this.emit('>');
         break;
       case ts.SyntaxKind.FunctionType:
-        this.emit('dynamic /*');
-        this.emit(node.getText());
-        this.emit('*/');
+        this.reportError(node, 'Inline function type declarations not supported. ' +
```

Comment:
This affects our user base and developers, so I think we should discuss this in a larger group, and not just declare facts by putting code in. Let's discuss what we want here in the standup.

I think if we enable the parameter function types, we're covering most of our use cases.

Also, is there any intention in Dart to support general function types?